RIck and Morty/View/Character/CharacterViewController.swift

```diff
@@ -19,15 +19,15 @@ class CharacterViewController: UIViewController {
   @IBOutlet weak var locationLabel: UILabel!
   @IBOutlet weak var imageView: UIImageView!
   
-  var character: CharacterModel?
+  private let viewModel: CharacterViewModel
```

Comment:
this might seem like a lot of overhead just to set some strings but if we think in terms of architecture, the view should never talk to models directly. This specific case is very simple but in real world applications the logic of each screen tends to grow and  by having the view models orchestrating the data flows from views down to network request we reduce the clutter in view controllers